src/components/FormItem.vue

```diff
@@ -9,10 +9,12 @@
     >
         <div class="form-item__field">
             <slot name="prepend"></slot>
+
```

Comment:
musime sa porozpravat o tych tvojich prazdnych riadkoch :D

Reply:
Resolved in 41869c9